examples/resources/datadog_sensitive_data_scanner_rule/resource.tf

```diff
@@ -23,6 +23,10 @@ resource "datadog_sensitive_data_scanner_rule" "myrule" {
     replacement_string = ""
     type               = "hash"
   }
+  included_keyword_configuration {
+    keywords        = ["cc", "credit card"]
+    character_count = 25
```

Comment:
Same here
```suggestion
    character_count = 30
```

Reply:
[Character count to 30 in the examples](https://github.com/DataDog/terraform-provider-datadog/pull/2238/commits/161d4627e84715fed617fc37114a946ef2d87326)